content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md

```diff
@@ -143,21 +142,39 @@ Once you have a Linux-based Kubernetes master node you are ready to choose a net
     kubectl get pods --all-namespaces
     ```
 
-    ![alt_text](../flannel-master-kubeclt-get-pods.png "flannel master kubectl get pods screen capture")
+    The output looks like as follows:
+
+    ```shell
+    NAMESPACE     NAME                                      READY        STATUS    RESTARTS   AGE
+    kube-system   etcd-flannel-master                       1/1          Running   0          1m
+    kube-system   kube-apiserver-flannel-master             1/1          Running   0          1m
+    kube-system   kube-controller-manager-flannel-master    1/1          Running   0          1m
+    kube-system   kube-dns-86f4d74b45-hcx8x                 3/3          Running   0          12m
+    kube-system   kube-flannel-ds-54954                     1/1          Running   0          1m
+    kube-system   kube-proxy-Zjlxz                          1/1          Running   0          1m
+    kube-system   kube-scheduler-flannel-master             1/1          Running   0          1m
+    ```
 
     Verify that the Flannel DaemonSet has the NodeSelector applied.
 
     ```bash
     kubectl get ds -n kube-system
     ```
 
-    ![alt_text](../flannel-master-kubectl-get-ds.png "flannel master kubectl get ds screen capture")
+    The output looks like as follows. The NodeSelector `beta.kubernetes.io/os=linux` is applied.
+
+    ```shell
```

Comment:
(same thing - this isn't a shell script).

Reply:
Okay, I'll apply.